website/project/model.py

```diff
@@ -604,6 +604,7 @@ class Node(GuidStoredObject, AddonModelMixin):
     files_versions = fields.DictionaryField()
     wiki_pages_current = fields.DictionaryField()
     wiki_pages_versions = fields.DictionaryField()
+    wiki_private_uuids = fields.DictionaryField()
```

Comment:
Can add a description and document the format of this field (e.g. `# {<page_name>: <sharejs_id>}`?